src/cdk/platform/platform.ts

```diff
@@ -59,4 +61,11 @@ export class Platform {
   // this and just place the Safari keyword in the userAgent. To be more safe about Safari every
   // Safari browser should also use Webkit as its layout engine.
   SAFARI: boolean = this.isBrowser && /safari/i.test(navigator.userAgent) && this.WEBKIT;
+
+  /**
+   * @deletion-target v7.0.0 remove optional decorator
+   */
+  constructor(@Optional() @Inject(PLATFORM_ID) private _platformId: Object) {
```

Comment:
This still doesn't make it a non-breaking change. You also have to mark the `_platformId` as an optional param by adding a question mark.

Reply:
Thought I did, my mistake. Will add, thanks for the catch